billing/v1/billing.proto

```diff
@@ -475,6 +475,18 @@ message ListBillingGroupsRequest {
   // For example, When Aws and Gcp to be filtered vendors=aws,gcp
   // If you want to get all vendors, not set parameter.
   string vendors = 2;
+  
+  // Optional. Filter BillingGroups by its tags. Format tags=key1:val1,key2:val2
+  repeated BillingGroupsTag tags = 3;
```

Comment:
@capoyon I think this idea is good so we don't need to add another field in case we want to support more filters but I think the term `tag` can be confusing. API users might think that we can tag Billing Groups when we actually don't allow it right now. We only filter by type. My suggestion is just use the word `filters` instead of `tag`.
Also, since this is a GET method, can you make sure if colon is allowed in the query string?